### PR TITLE
build: bump Rust version

### PR DESCRIPTION
**Pull request by stuartcarnie**
- 🚀 Bump Rust version to 1.70.0 0fc859f616f1c67ba634902f58e9a9080fda8a53
- 🧠 `clippy` is smarter via 28166006a8b9c91f0b5c10958e9fdd1efda3de41 b6f298f da682d8
- 🧹 `rustfmt` via a4371254ebf833490743d5ff66e197ced179b1c2

### CODE REVIEW

File: garbage_collector/src/server.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (alamb) [inline garbage_collector/src/server.rs:1]: Not that this PR made this worse, but this file seems unecessary 🤔

**Review by alamb — APPROVED** [review]
Looks great to me!

File: garbage_collector/src/server.rs (hunk 1 above)
Comment (stuartcarnie) [reply garbage_collector/src/server.rs]: It is not included in the `lib.rs`, so I removed it. If someone was actively working on it, would be a minimal burden to re-add.

Comment (alamb) [reply garbage_collector/src/server.rs]: cc @philjb

Comment (philjb) [reply garbage_collector/src/server.rs]: thanks for the heads up! not mine or at least not intentionally mine!

**Pull request merged**